files/image_config/platform/rc.local

```diff
@@ -319,6 +319,15 @@ if [ -f $FIRST_BOOT_FILE ]; then
         mv /host/grub.cfg /host/grub/grub.cfg
     fi
 
+    if `dpkg -l kdump-tools > /dev/null 2>&1`;
+    then
+	sed -i 's/[^M] quiet/ crashkernel=256M quiet/' /host/grub/grub.cfg
+
+	kdump-config unload
+	kdump-config symlinks `uname -r`
+	kdump-config load
```

Comment:
this only works for the next reboot. can we get a solution that works for current boot?

Reply:
agree but this is an known issue, it is hard to get it work with first  reboot due to the nature of the grub, and changes are made to the new image,while the sonic_installer used in the upgrade actually from the previous image.

Reply:
you can modify the installer/x86_64/install.sh file

Reply:
thanks for pointing out this , I move grub.cfg change to install.sh and make it optional depends on build option